tests/jballerina-bstring-unit-test/src/test/resources/test-src/types/finite/finite-type.bal

```diff
@@ -461,6 +461,31 @@ function testFiniteTypesWithDiscriminatedMembers() returns [any, any, any, any,
     return [a, b, c, d, e];
 }
 
+type d +3|+5;
+
+function testFiniteTypesWithPositiveIntegers() {
+    d n = +3;
+    d comparator = +3;
+    if (n == comparator) {
+       n = +5;
+    }
+    if (n != 5){
+        panic error("Type mismatch");
```

Comment:
Best practices for errors
- as the error reason use meaningful simple strings with no spaces
- as the message field give additional details about what caused the error

See https://github.com/ballerina-platform/ballerina-lang/blob/7fa2ee515984b42c1281a43621753838474f6907/tests/jballerina-bstring-unit-test/src/test/resources/test-src/expressions/listconstructor/list_constructor_infer_type.bal#L64 for an example.

Reply:
Fixed